core/expressionParser/helpers/functionImplementations.js

```diff
@@ -5,6 +5,7 @@ import { functionNames } from './functions'
 
 export const functionImplementations = {
   [functionNames.avg]: A.identity,
+  [functionNames.categoryItemProp]: () => null, // To be implemented by evaluator sub classes
```

Comment:
Add here (maybe) TODO:
So it can be found easier

Unless of course, you remember this :)

Reply:
the fact is that the implementation is empty; it can be estended by subclasses.
Anyway, we willl get rid of this implementation of the expression evaluator and use the one in arena-core, it's better!